lib/rules/jsx-no-bind.js

```diff
@@ -5,13 +5,25 @@
  */
 'use strict';
 
+var Components = require('../util/Components');
 var propName = require('jsx-ast-utils/propName');
 
 // -----------------------------------------------------------------------------
 // Rule Definition
 // -----------------------------------------------------------------------------
 
-module.exports = function(context) {
+function renderReturnsJSX(node, utils) {
```

Comment:
I agree. Sounds like we should move it to the `utils` provided by the `Components` utility?  What should it be named? There is already a `isReturningJSX` that does the bulk of this work but this deals with finding the `ReturnStatement`